spec/bowling_game_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'bowling_game'
+
+describe BowlingGame do
+  subject(:game) { described_class.new }
+
+  describe '.play' do
+    it "should allow game to respond to play" do
+      expect(BowlingGame.new).to be_a(BowlingGame)
+    end
+  end
+
+  describe '#bowl' do
+    it "should allow the user to input the score for a frame" do
+      expect(game).to respond_to(:roll).with(1).argument
+    end
+
+    it "can roll a gutter game" do
+      BowlingGame::MAX_ROLLS.times { game.roll(0) }
+      expect(game.score).to eq(0)
+    end
+
+    it "can roll all 4s" do
+      BowlingGame::MAX_ROLLS.times { game.roll(4) }
+      expect(game.score).to eq(80)
+    end
+
+    it "can roll a spare" do
+      game.roll(5)
```

Comment:
Would a spare not work if you put this in instead?:
`game.roll(9)`
`game.roll(1)`

Reply:
Wait I think I understand, why there is the extra `game.roll(3)`, no worries, me not reading ahead.